### PR TITLE
Clean up flyteadmin proto definitions and comments.

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
Clean up flyteadmin proto definitions and comments.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_

### CODE REVIEW

File: protos/flyteidl/admin/execution.proto [hunk 1]
```diff
@@ -276,9 +282,9 @@ message WorkflowExecutionGetDataRequest {
 // Response structure for WorkflowExecutionGetDataRequest which contains inputs and outputs for an execution.
 message WorkflowExecutionGetDataResponse {
     // Signed url to fetch a core.LiteralMap of execution outputs.
-    UrlBlob outputs = 1;
+    UrlBlob outputs = 1 [deprecated = true];
```
Comment (EngHabu) [inline protos/flyteidl/admin/execution.proto:285]:
```suggestion
    // Deprecated: Please use full_outputs instead.
    UrlBlob outputs = 1 [deprecated = true];
```

File: protos/flyteidl/admin/execution.proto [hunk 2]
```diff
@@ -276,9 +282,9 @@ message WorkflowExecutionGetDataRequest {
...
     // Signed url to fetch a core.LiteralMap of execution inputs.
-    UrlBlob inputs = 2;
+    UrlBlob inputs = 2 [deprecated = true];
```
Comment (EngHabu) [inline protos/flyteidl/admin/execution.proto:287]:
```suggestion
    // Deprecated: Please use full_inputs instead.
    UrlBlob inputs = 2 [deprecated = true];
```

File: protos/flyteidl/admin/node_execution.proto [hunk 3]
```diff
@@ -171,10 +185,10 @@ message NodeExecutionGetDataRequest {
 // Response structure for NodeExecutionGetDataRequest which contains inputs and outputs for a node execution.
 message NodeExecutionGetDataResponse {
     // Signed url to fetch a core.LiteralMap of node execution inputs.
-    UrlBlob inputs                             = 1;
+    UrlBlob inputs                             = 1  [deprecated = true];
```
Comment (EngHabu) [inline protos/flyteidl/admin/node_execution.proto:188]:
```suggestion
    // Deprecated: Please use full_inputs instead.
    UrlBlob inputs                             = 1  [deprecated = true];
```

File: protos/flyteidl/admin/node_execution.proto [hunk 4]
```diff
@@ -171,10 +185,10 @@ message NodeExecutionGetDataRequest {
...
 
     // Signed url to fetch a core.LiteralMap of node execution outputs.
-    UrlBlob outputs                            = 2;
+    UrlBlob outputs                            = 2  [deprecated = true];
```
Comment (EngHabu) [inline protos/flyteidl/admin/node_execution.proto:191]:
```suggestion
    // Deprecated: Please use full_outputs instead.
    UrlBlob outputs                            = 2  [deprecated = true];
```

**Review by EngHabu — APPROVED** [review]

File: protos/flyteidl/admin/workflow_attributes.proto [hunk 5]
```diff
@@ -46,14 +52,19 @@ message WorkflowAttributesGetResponse {
...
 
     // Unique domain id which this set of attributes references.
+    // +required
     string domain                  = 2;
 
     // Workflow name which this set of attributes references.
+    // +required
     string workflow                = 3;
 
+    // Which type of matchable attributes to delete.
```
Comment (kumare3) [inline protos/flyteidl/admin/workflow_attributes.proto:66]: lets put a reference to MatchableAttributes and in that proto explain what Matchable attributes really is

File: protos/flyteidl/admin/workflow_attributes.proto [hunk 6]
```diff
@@ -46,14 +52,19 @@ message WorkflowAttributesGetResponse {
 
 message WorkflowAttributesDeleteRequest {
```
Comment (kumare3) [inline protos/flyteidl/admin/workflow_attributes.proto:53]: Comment for this proto?

File: protos/flyteidl/admin/task_execution.proto [hunk 7]
```diff
@@ -108,19 +111,20 @@ message TaskExecutionClosure {
...
 message TaskExecutionGetDataResponse {
     // Signed url to fetch a core.LiteralMap of task execution inputs.
-    UrlBlob inputs                             = 1;
+    UrlBlob inputs                             = 1 [deprecated = true];
 
     // Signed url to fetch a core.LiteralMap of task execution outputs.
-    UrlBlob outputs                            = 2;
+    UrlBlob outputs                            = 2 [deprecated = true];
 
     // Optional, full_inputs will only be populated if they are under a configured size threshold.
```
Comment (kumare3) [inline protos/flyteidl/admin/task_execution.proto:129]: this is not optional anymore right?

File: protos/flyteidl/admin/project.proto [hunk 8]
```diff
@@ -43,6 +33,17 @@ message Project {
     // tag projects with ownership information.
     Labels labels = 5;
 
+    // The state of the project is used to control its visibility in the UI and validity.
+    enum ProjectState {
```
Comment (kumare3) [inline protos/flyteidl/admin/project.proto:37]: is this new?

File: protos/flyteidl/admin/node_execution.proto [hunk 9]
```diff
@@ -171,10 +185,10 @@ message NodeExecutionGetDataRequest {
...
 
     // Optional, full_inputs will only be populated if they are under a configured size threshold.
```
Comment (kumare3) [inline protos/flyteidl/admin/node_execution.proto:193]: is this really optional now that urlblob is deprecated?

File: protos/flyteidl/admin/project.proto (hunk 8 above)
Comment (katrogan) [reply protos/flyteidl/admin/project.proto:37]: no

File: protos/flyteidl/admin/workflow_attributes.proto (hunk 5 above)
Comment (katrogan) [reply protos/flyteidl/admin/workflow_attributes.proto:70]: sure, done

File: protos/flyteidl/admin/task_execution.proto (hunk 7 above)
Comment (katrogan) [reply protos/flyteidl/admin/task_execution.proto]: updated

File: protos/flyteidl/admin/node_execution.proto (hunk 9 above)
Comment (katrogan) [reply protos/flyteidl/admin/node_execution.proto]: updated

File: protos/flyteidl/admin/workflow_attributes.proto (hunk 6 above)
Comment (katrogan) [reply protos/flyteidl/admin/workflow_attributes.proto:57]: updated

File: protos/flyteidl/admin/node_execution.proto (hunk 4 above)
Comment (katrogan) [reply protos/flyteidl/admin/node_execution.proto:193]: updated

File: protos/flyteidl/admin/node_execution.proto (hunk 3 above)
Comment (katrogan) [reply protos/flyteidl/admin/node_execution.proto:189]: updated

File: protos/flyteidl/admin/execution.proto (hunk 2 above)
Comment (katrogan) [reply protos/flyteidl/admin/execution.proto:290]: updated

File: protos/flyteidl/admin/execution.proto (hunk 1 above)
Comment (katrogan) [reply protos/flyteidl/admin/execution.proto:286]: updated

**Comment by katrogan** [comment]
PTAL @EngHabu @kumare3

File: protos/flyteidl/admin/project.proto (hunk 8 above)
Comment (kumare3) [reply protos/flyteidl/admin/project.proto:37]: ohh sorry, i see it was moved

File: protos/flyteidl/admin/node_execution.proto [hunk 10]
```diff
@@ -171,15 +185,17 @@ message NodeExecutionGetDataRequest {
...
+    // Deprecated: Please use full_inputs instead.
+    UrlBlob inputs                             = 1  [deprecated = true];
 
     // Signed url to fetch a core.LiteralMap of node execution outputs.
-    UrlBlob outputs                            = 2;
+    // Deprecated: Please use full_outputs instead.
+    UrlBlob outputs                            = 2  [deprecated = true];
 
-    // Optional, full_inputs will only be populated if they are under a configured size threshold.
+    // Full_inputs will only be populated if they are under a configured size threshold.
```
Comment (kumare3) [inline protos/flyteidl/admin/node_execution.proto:195]: what if they are over the size threshold? I think that will cause runtime failures right?

Comment (katrogan) [reply protos/flyteidl/admin/node_execution.proto:195]: @kumare3 perhaps, the input and output URIs are always available in the closure

what do you think is right here? always return inline and remove the size check altogether?

**Review by kumare3 — APPROVED** [review]

**Review by EngHabu — APPROVED** [review]

**Pull request merged**